kubernetes/config/worker.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
What's with this rename? The example deployment is still based on Ubuntu 16.04, right? People may want to add other kinds of workers to their cluster as well, so just calling it 'worker' is too generic.